tests/benchmarks/json_set_ResultSet.totalResultsAvailable_1_jsonsl-yahoo2_json.yml

```diff
@@ -14,4 +14,4 @@ clientconfig:
     - requests: 5000000
     - threads: 2
     - pipeline: 1
-    - command: 'JSON.GET jsonsl-yahoo2 ResultSet.totalResultsAvailable 1'
+    - command: 'JSON.SET jsonsl-yahoo2 $.ResultSet.totalResultsAvailable 1'
```

Comment:
@filipecosta90 it seems like a mistake should it be JSON.GET or SET?

Reply:
it should be json.set. This is a fix

Reply:
@filipecosta90 but the description still says `JSON.GET jsonsl-yahoo2`